### PR TITLE
Built discovery server container with unsigned packages. 

### PR DESCRIPTION
**Pull request by myersCody**
# Scratch build setup
RHEL8 add-scratch-repo command:
```
make podman-test-scratch repo=brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/discovery/discovery-server-rhel8:discovery-0.9-rhel-8-containers-candidate-75921-20190729180526
```
## Verification on RHEL 8

- [x] make setup-8
- [x] make import-8
- [x] make scratch-8
- [x] make init-test-8
- [x] make podman-test-scratch repo=`<repo>`

### CODE REVIEW

File: rcm_data/discovery-server-container/Makefile [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+	cd ../; python3 install_official.py dep_pkgs=discovery os_ver=rhel8
+	sudo podman pull $(repo) --tls-verify=false
+	mkdir -p ~/quipucords
+	cd ~/quipucords; echo sshkeys log qpc-data data | xargs mkdir
+	sudo podman pod create --name qpc_pod -p 0.0.0.0:9443:443 -p 5432
+	sudo podman run --name qpc-db --pod qpc_pod -e POSTGRES_USER="postgres" -e POSTGRES_PASSWORD="password" -v qpc-data:/var/lib/postgresql/data -dt postgres
+	sudo podman run --name quipucords --net host --pod qpc_pod -e "USE_SUPERVISORD=true" -e "QPC_DBMS_USER=postgres" -e "QPC_DBMS_PASSWORD=password" -e "ANSIBLE_LOG_LEVEL=$(ans_log_level)" -v ~/quipucords/sshkeys:/sshkeys:z -v ~/quipucords/data:/var/data:z -v ~/quipucords/log:/var/log:z -dt $(repo)
+	dsc server config --host 0.0.0.0 --port 443
+	dsc server login
+	read -p "You may want to `make logs` in another window. Press enter to continue:  "
```
[the remark this answers is not in the document]
Comment (myersCody) [reply rcm_data/discovery-server-container/Makefile]: I think I may just remove it. Now that I think about it logs are not required to test it.

File: README.rst [hunk 2]
```diff
@@ -76,6 +76,51 @@ Testing for RHEL7 or RHEL8
 
     make test-scratch-7
 
+Developing Containers for RHEL8
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow for developing a container is generally the same. Unfortunately, due to the manner in which the containers are built there is no functionality to locally mock the container. Therefore, each test needs to be run as a scratch build. Additionally, in order to run a scratch build you have to commit any changes to the git dist repo.
+
+1. Setup rhpkg, brew, and git.::
```
Comment (abaiken) [inline README.rst]: Remove the `.` before the `::`.

File: README.rst [hunk 3]
```diff
@@ -76,6 +76,51 @@ Testing for RHEL7 or RHEL8
...
+
+6. Clean the development environment with::
+
+    make clean-8
+
+Testing Containers for RHEL8
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create & access the testing vagrant machine::
+
```
Comment (abaiken) [inline README.rst]: I think you're missing `make test-8` under this step.

File: README.rst [hunk 4]
```diff
@@ -76,6 +76,51 @@ Testing for RHEL7 or RHEL8
...
+    sudo su
+    cd /rcm_data/discovery-server-container/
+    make init-test-8
+
+2. To test the stratch container repo run the following command::
```
Comment (abaiken) [inline README.rst]: change `stratch` to `scratch`

File: rcm_data/discovery-server-container/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+# Container Deps
+RUN yum -y install /usr/bin/ssh
+RUN yum -y install sshpass
+RUN yum -y install which
+RUN yum -y install procps
+RUN yum -y install python3-tools
+RUN yum -y install discovery-server
+RUN yum clean all
+
+#RUN mkdir -p /app
```
Comment (abaiken) [inline rcm_data/discovery-server-container/Dockerfile]: do we need these commented out commands?

File: rcm_data/discovery-server-container/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+# Create a symbolic link to discover-server
+# RUN  ln -s  /usr/lib/python3.6/site-packages/quipucords /app/
```
Comment (abaiken) [inline rcm_data/discovery-server-container/Dockerfile]: Same ? as above.

Comment (myersCody) [reply rcm_data/discovery-server-container/Dockerfile]: I am not certain to if I will need this for 0.9.1 or not.

**Pull request merged**